frontend/src/styles.scss

```
@@ -1,5 +1,17 @@
 @use "sass:meta";
 @import "primeicons/primeicons.css";
+
+@font-face {
+  font-family: "primeicons";
```

Stylelint (17.7.0)

[error] Expected no quotes around "primeicons" (font-family-name-quotes)

(font-family-name-quotes)